packages/contracts/utils/event.ts

```diff
@@ -21,7 +21,7 @@ export async function findEventTopicLog(
   const topic = iface.getEventTopic(eventName);
   const log = receipt.logs.find(x => x.topics[0] === topic);
   if (!log) {
-    throw new Error(`No logs found for this event ${eventName} topic.`);
+    throw new Error(`No logs found for the topic of event "${eventName}".`);
   }
   return iface.parseLog(log);
```

Comment:
couldn't `findEventTopicLog` require a generic as `findEvent` did so we have proper typing in the returned log?

Reply:
Improved by a9e40fba4c9adcb680cf97e79c497fdb9300dbd6

so we get these type hints now:

![Bildschirmfoto 2023-07-20 um 15 19 11](https://github.com/aragon/osx/assets/20623991/486ec778-3f94-4d19-a978-b55547e69945)


![Bildschirmfoto 2023-07-20 um 15 19 26](https://github.com/aragon/osx/assets/20623991/f049b300-db12-4ff6-8836-491046bb1516)